content/js/getting-started.md

```diff
@@ -0,0 +1,46 @@
+<a name="sign-up-portal"></a>
+## Signing up at Portal
+
+To start using the Skygear JS SDK, you need to register your account and
+application at the [Skygear Developer Portal](https://portal.skygear.io/signup).
+You will be able to monitor your Skygear server and acquire information, such
+as your `Server EndPoint` and `API Key`, on the portal page.
```

Comment:
- Server endpoint and API key are not codes. Also, "endpoint" is (can be) one word.
- Move "on the portal page" to the beginning of the sentence like "In the portal, "